docs/team-process.md

```diff
@@ -167,6 +167,7 @@ Below is a list of dependencies that are locked down due to known issues with se
 
 - It'd be good to keep an eye on `s3rver` for when it exceeds 3.7.1. We have a patch in place for called `s3rver+3.7.1.patch` in order to address the high severity issue exposed by `s3rver`'s dependency on `busboy` 0.3.1, which relies on `dicer` that actually has the [security issue](https://github.com/advisories/GHSA-wm7h-9275-46v2). Unfortunately, `busboy` >0.3.1, aka ^1.0.0, is incompatible with s3rver which is why there's a patch in place to make it compatible.
 
-#### Validating Updates
--  After changes are made to any dependencies, deploy to an exp environment to verify that all tests pass!
-    - Be sure the deploy [runs a migration](./additional-resources/blue-green-migration.md#manual-migration-steps) to verify the updates do not affect the migration workflow.
```

Comment:
I think we should still keep this guidance, yes?  This is still a good thing to do.

Reply:
Ah, this is something Josh was cleaning up since I had moved it to line 152 inside the Library Update Steps, so this one is a duplicate now